multichain-testing/Makefile

```diff
@@ -61,7 +61,7 @@ setup-kind:
 
 .PHONY: clean-kind
 clean-kind:
-	kind delete cluster --name $(KIND_CLUSTER)
+	-kind delete cluster --name $(KIND_CLUSTER)
```

Comment:
Is this intentional? What does it do?

Reply:
Triggers a silent failure in case the target fails avoiding disruptively halting the Makefile.

Reply:
TIL!

Is this PR still draft?

Reply:
Oops. Let me make it R4R.